website/docs/migrating-from-v1-to-v2.md

```diff
@@ -504,3 +506,154 @@ You might want to refer to our migration PRs for [Create React App](https://gith
 ## Support
 
 For any questions, you can ask in the [`#docusaurus-1-to-2-migration` Discord channel](https://discordapp.com/invite/kYaNd6V). Feel free to tag us ([@yangshun](https://github.com/yangshun) and [@endiliey](https://github.com/endiliey)) in any migration PRs if you would like us to have a look.
+
+## Versioned Site
+
+> :warning: _This section is a work in progress._
+
+> ⚠️ Although we've implemented docs versioning since 2.0.0-alpha.37, we'd like to test it out for v2 users first before we recommend v1 users to migrate to v2. There are some changes in how v2 versioning works compared to v1. In the future, we might create a script to migrate your versioned docs easier. However, if you are adventurous enough to manually migrate, feel free to do so. Be warned though, the manual migration requires lot of work.
+
+## Changes from v1
+- Read up https://v2.docusaurus.io/blog/2018/09/11/Towards-Docusaurus-2#versioning first for reasoning on v1's problem
+
+### Migrate your versioned_docs frontmatter
+- Unlike v1, The markdown header for each versioned doc is no longer altered by using `version-${version}-${original_id}` as the value for the actual id field. See scenario below for better explanation.
+
+Example, you have a `docs/hello.md`. 
+```md
+---
+id: hello
+title: Hello, World !
+---
+Hi, Endilie here :)
+```
+
+When you cut a new version 1.0.0
+
+In Docusaurus v1, `website/versioned_docs/version-1.0.0/hello.md` looks like this
+
+```md
+---
+id: version-1.0.0-hello
+title: Hello, World !
+original_id: hello
+---
+Hi, Endilie here :)
+```
+
+In comparison, Docusaurus 2 `website/versioned_docs/version-1.0.0/hello.md` looks like this (exactly same as original)
+
+```md
+---
+id: hello
+title: Hello, World !
+---
+Hi, Endilie here :)
+```
+
+Since we're going for snapshot and allows people to move (and edit) docs easily inside version. The id frontmatter is no longer altered, it's gonna be keep as it is. But internally it is set as `version-${version}/${id}`
```

Comment:
- snapshot and allow
- The id frontmatter is no longer altered and will remain the same cross versions. Internally, ...